### PR TITLE
Add Mortgage_calculator to Jenkins

### PR DESCRIPTION
**Pull request by osharpe**
https://moneyadviceservice.tpondemand.com/entity/8726

Changes required in order to add this repo in Jenkins for CI

In addition, the test group in Gemfile has been updated: to include tzinfo-data and brakeman gems.

Slight modification also required to load Angular service files in karma.conf, as I faced issues getting the Karma tests to run correctly in a Docker build.

### CODE REVIEW

**Comment by tomas-stefano** [comment]
👍

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+#Install PhantomJs
+RUN curl -L -O https://bitbucket.org/ariya/phantomjs/downloads/phantomjs-${PHANTOMJS_VERSION}-linux-x86_64.tar.bz2
+RUN tar jxf phantomjs-${PHANTOMJS_VERSION}-linux-x86_64.tar.bz2 && \
+    mv phantomjs-${PHANTOMJS_VERSION}-linux-x86_64 /usr/local/share && \
+    ln -sf /usr/local/share/phantomjs-${PHANTOMJS_VERSION}-linux-x86_64/bin/phantomjs /usr/local/bin
+
+RUN mkdir -p /var/tmp/bin/node /var/tmp/bin/ruby /var/tmp/downloads /var/tmp/app
+#Install Ruby
+COPY .ruby-version .ruby-version
```
Comment (willhall88) [inline Dockerfile]: not sure I understand this line?

**Comment by willhall88** [comment]
👍  great work. looking forward to seeing it running

File: Dockerfile (hunk 1 above)
Comment (osharpe) [reply Dockerfile]: It does look a bit strange but this copies from the workspace in Jenkins into the Docker volume.

**Pull request merged**